cc-mt-service/src/app.ts

```diff
@@ -1,7 +1,6 @@
 import express from 'express';
-import axios from 'axios';
 import dotenv from 'dotenv';
-import qs from 'qs';
+import router from './routes/mtServiceRoutes';
 
 dotenv.config();
 
```

Comment:
I wasn't able to comment on the lines directly, but in this file (app.ts) the variables cId, cSecret and scope are now unused and can be removed.